sys/vm/vm_cheri_revoke.c

```diff
@@ -823,17 +821,6 @@ vm_cheri_revoke_pass(const struct vm_cheri_revoke_cookie *crc, int flags)
 		}
 	}
 
-	/*
-	 * We could do this to ensure that the TLB has no stale CLG entries,
-	 * but that's not really requisite: they'll either get pushed out,
-	 * we'll either trap on them belatedly and fix them up
-	 * (PMAP_CAPLOADGEN_ALREADY), or they'll get shot down when we
-	 * increment the CLG for the *next* pass (while we've got the world
-	 * stopped.)
-	 *
-	pmap_invalidate_all(pmap);
-	 */
```

Comment:
Is this detail we want to be losing?

Reply:
I've gone ahead and restored it.

Reply:
I would format it something like:

```
	 *   pmap_invalidate_all(pmap);
```
though